tests/unit/lms/security_test.py

```diff
@@ -347,21 +347,24 @@ def test_permits_denied_with_validation_error(self, pyramid_request):
         assert is_allowed == DeniedWithException(validation_error)
 
     @pytest.mark.parametrize(
-        "lti_user,expected_userid",
+        "user_id,application_instance_id,expected_userid",
         [
-            (None, None),
-            (
-                factories.LTIUser(
```

Comment:
Using a factory here outside a method prevents it from using sub factories (it doesn't have access to the DB session).

This is the only place where we used that technique for any factory.